src/tests/ftest/recovery/ddb.py

```diff
@@ -148,10 +148,12 @@ def test_recovery_ddb_ls(self):
 
         # Find the vos file name. e.g., /mnt/daos0/<pool_uuid>/vos-0.
         scm_mount = self.server_managers[0].get_config_value("scm_mount")
+        vos_file = self.get_vos_file_path(pool=self.pool)
+        if vos_file is None:
+            self.fail("vos file wasn't found in {}/{}".format(scm_mount, self.pool.uuid.lower()))
```

Comment:
Since `get_vos_file_path` will log the path, I think it's okay to simplify these to something like
```suggestion
        if vos_file is None:
            self.fail("Failed to find vos file")
```
Thoughts?

Reply:
`get_vos_file_path` logs the path like this:
```
08:15:14 DEBUG| Running on wolf-146 with a 120 second timeout: sudo ls /mnt/daos0/83bc94e4-ba72-4965-92a8-a1dacbad4248
08:15:15 DEBUG|   wolf-146 (rc=0):
08:15:15 DEBUG|     rdb-pool
08:15:15 DEBUG|     vos-0
08:15:15 DEBUG|     vos-1
08:15:15 DEBUG|     vos-2
08:15:15 DEBUG|     vos-3
08:15:15 INFO | vos_file: vos-0
08:15:15 DEBUG| Updated param vos_path => /mnt/daos0/83bc94e4-ba72-4965-92a8-a1dacbad4248/vos-0
```
Although it's printed, I think it would be more helpful if the error message contains the path so that it requires one less thing to search. Also it doesn't reduce the number of lines of code.